src/main/java/org/sopt/lequuServer/domain/book/service/BookService.java

```diff
@@ -0,0 +1,79 @@
+package org.sopt.lequuServer.domain.book.service;
+
+import lombok.RequiredArgsConstructor;
+import org.sopt.lequuServer.domain.book.dto.request.BookCreateRequest;
+import org.sopt.lequuServer.domain.book.dto.response.BookCreateResponse;
+import org.sopt.lequuServer.domain.book.model.Book;
+import org.sopt.lequuServer.domain.book.repository.BookRepository;
+import org.sopt.lequuServer.domain.member.model.Member;
+import org.sopt.lequuServer.domain.member.repository.MemberRepository;
+import org.sopt.lequuServer.global.auth.jwt.JwtProvider;
+import org.sopt.lequuServer.global.exception.enums.ErrorType;
+import org.sopt.lequuServer.global.exception.model.CustomException;
+import org.sopt.lequuServer.global.s3.service.S3Service;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.sopt.lequuServer.global.s3.enums.ImageFolderName.BOOK_FAVORITE_IMAGE_FOLDER_NAME;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class BookService {
+
+    private final BookRepository bookRepository;
+    private final MemberRepository memberRepository;
+    private final S3Service s3Service;
+    private final JwtProvider jwtProvider;
+
+    @Transactional
+    public BookCreateResponse createBook(BookCreateRequest request, String authorization) {
+
+        /**
+         * 요청을 한 유저를 가장 먼저 검증하고
+         * 검증이 된 유저에 한해서 동작을 할 수 있도록
+         * 로직을 순서대로 작성해야함
+         * 청을 한 유저 검증 -> Filter에서 다 해줌
+         */
+
+        // Bearer을 제외한 토큰 string만 뺴오는곳
+        String token = authorization.split(" ")[1];   // Bearer는 [0]에 담긴 것, 토큰 스트링이 [1]에 담긴 것
+
+        // 유효한 토큰이 가지고 있는 유저 정보가 올바른지
+        Long userId = jwtProvider.getUserFromJwt(token);
+        Optional<Member> findById = memberRepository.findById(userId);
+
+        // 전송된 토큰으로 식별되는 유저가 없을 경우
+        if (findById.isEmpty()) {
+            throw new CustomException(ErrorType.NOT_FOUND_MEMBER_ERROR);
+        }
+
+        // optional 객체 안에 있는 실제 User 객체를 꺼냄
+        Member member = findById.get();
```

Comment:
[P3] 혹은 이 로직 전체를 Repository의 default method로 쓰는 것도 좋아보입니다!
깔씀성의 측면에서...?
```suggestion
        Member findById = memberRepository.getMemberById(userId);
        ---
        // 아래는 Repository단 예시입니다.
        default Member getMemberById(Long id) {
            return this.findById(id).orElseThrow(
                () -> new NotFoundException(ErrorStatus.NOT_FOUND_USER_EXCEPTION,
                    ErrorStatus.NOT_FOUND_USER_EXCEPTION.getMessage()));
        }
```

Reply:
오 이런 방법이 있군요 !! 감사합니당 ~~~~